### ISSUE TITLE
Suggestion for email notifications

### ISSUE THREAD

**Issue by centminmod** [certbot/certbot#741]
For email notifications for client failures, sucess/failed auto renewal etc. Is there support for more than one email address ? If there isn't support, I suggest to add support for multiple email addresses.

This way folks can setup email notifications to multiple folks or multiple service based notifications. For example with pushover.net you can setup an email address at yourUSERkey+devicename+p1@api.pushover.net  for priority 1 highest priority push notifications to mobile and tablet devices. Handy to have :)

This is based on discussions regarding renewal process at https://community.letsencrypt.org/t/client-auto-renewals-and-failure-notifications/594

**Comment by trinopoty** [comment]
It seems that email notifications are disabled atm. Also, it would be better to support SMTP with user provided config alongside LMTP.

**Comment by schoen** [comment]
@jsha, does Boulder support multiple addresses if the client submits them with a registration?

@trinopoty, currently all of the notifications are provided by the certificate authority side, and none from the Certbot side (differently from what we envisioned when we first started the project). Therefore, this issue only relates to the information that Certbot submits to the CA when registering or updating an account.
